rpmlint/checks/FileDigestCheck.py

```diff
@@ -74,8 +75,11 @@ def parse_content(self):
                     # skip empty lines
                     continue
                 elif line_nr == 0 and stripped.startswith('#!'):
-                    # keep shebang lines instact
-                    pass
+                    # keep shebang lines mostly intact,
+                    # but ignore minor interpreter versions
+                    shebang = re.sub(r'\bpython3\.\d+\b', 'python3', line)
+                    yield shebang.encode()
```

Comment:
could it make sense not to `yield` here another time, but to say `line = re.sub(...)` and continue using the `yield` after the if clauses?

That way any logic that is generally applied to lines stays in one place.